services/graylog/scripts/configure.py

```diff
@@ -233,11 +312,16 @@ def configure_log_retention(_session, _headers):
         print("Error setting up graylog syslog capturing.")
 
 
-def configure_alerts(_mail_notification_id, _session, _headers):
+def configure_alerts(
+    _session,
```

Comment:
I would annotate all your inputs/outputs so that static tools can assist you finding issues  ... specially since you do not have tests. a

Reply:
will do, thx